.github/workflows/sdk_build.yml

```diff
@@ -6,12 +6,12 @@ on:
     inputs:
       # Which version of Unity to use
       unity_version:
-        description: 'Unity version'
-        default: '2019.4.30f1'
+        description: 'Unity version [2017.4.40f1, 2018.4.36f1, 2019.4.32f1, 2020.3.21f1]'
+        default: '2019.4.32f1'
         required: true
       # Linux uses a different number for its versions
       linux_unity_version:
-        description: 'Linux Unity version'
+        description: 'Linux Unity version [2017.4.10f1, 2018.4.36f1, 2019.4.32f1, 2020.3.21f1]'
```

Comment:
What about add Unity version to the build_desktop job name?
build-desktop-${{matrix.os}}-${{ github.event.inputs.unity_version }}

Reply:
Done